<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../recursos.qrc">
    <normaloff>:/icons/assets/Peppa.png</normaloff>:/icons/assets/Peppa.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btnT1">
      <property name="text">
       <string>Tarea 1 - Switch</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnT2">
      <property name="text">
       <string>Tarea 2 - Editar Dinero</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnT3">
      <property name="text">
       <string>Tarea 3 - Símbolo de batería</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnT4">
      <property name="text">
       <string>Tarea 4 - Contador de carácteres</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPeppa"/>
  </widget>
  <action name="actionPeppa">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../recursos.qrc">
     <normaloff>:/icons/assets/Peppa.png</normaloff>:/icons/assets/Peppa.png</iconset>
   </property>
   <property name="text">
    <string>Peppa</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../recursos.qrc"/>
 </resources>
 <connections/>
</ui>
